scrapers/spiders/__init__.py

```diff
@@ -0,0 +1,21 @@
+from scrapy import Spider
+
+
+class BaseSpider(Spider):
+    def __init__(self,  **kwargs):
+        super().__init__(**kwargs)
+
+    def parse(self, response):
```

Comment:
Can you tell me a little of your thoughts about making the change from `start_requests()` in each spider to kicking things off with `parse()` in this `BaseSpider` class? I'm not sure I understand the benefits

Reply:
start_requests() - This function is invoked to initiate the initial requests. It should return/yield a Request object or None. If it returns a dictionary or an item, it can lead to errors in Scrapy. If the Request object does not have a specified callback function, Scrapy defaults to calling the `parse` function. 
In this reason, we can't return Jurisdiction item or Organization in the start_requests() function.

Reply:
https://github.com/scrapy/scrapy/blob/0e78acb65798a1eb4e55a472232e77d55e6c7cd5/scrapy/spiders/__init__.py#L68

Reply:
OK, that's helpful to know, thank you!